migrations/013-update-user-isAdmin.js

```diff
@@ -0,0 +1,14 @@
+const tableName = 'Users';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
```

Comment:
There's no need for a promise here since only one transaction in this migration.

Reply:
ok...updated it.